Change in Wazuh documentation Ansible (#4479)
wazuh/wazuh-documentation: source/deploying-with-ansible/guide/install-opendistro-server.rst

	ansible@ansible:/etc/ansible/roles/wazuh-ansible$ tree playbooks/

.. code-block:: none
	:class: output

	playbooks/
	├── wazuh-agent.yml
	├── wazuh-elastic_stack-distributed.yml
	├── wazuh-elastic_stack-single.yml
	├── wazuh-elastic.yml
	├── wazuh-kibana.yml
	├── wazuh-manager-oss.yml
	├── wazuh-manager.yml
	├── wazuh-odfe-production-ready.yml
	├── wazuh-odfe-single.yml
	├── wazuh-opendistro-kibana.yml
	└── wazuh-opendistro.yml

Using **Opendistro** and **Kibana** roles we will install and configure the Opendistro Stack server components, there are several variables we can use to customize the installation or configuration. To consult the default configuration go to this :ref:`section <wazuh_ansible_reference>`.

If we want to change the default configuration we can change the following files:
- ``/etc/ansible/roles/wazuh-ansible/roles/opendistro/opendistro-elasticsearch/defaults/main.yml``
- ``/etc/ansible/roles/wazuh-ansible/roles/opendistro/opendistro-kibana/defaults/main.yml``

We also can create another YAML file only with the content we want to change the configuration for each role. We can find more information here:

- :ref:`Opendistro <opendistro-elasticsearch>` role.

> - :ref:`Kibana <opendistro-kibana>` role.

Author: Done

          node1:
            name: node-1
            ip: <node-1 IP>
          node2:
            name: node-2
            ip: <node-2 IP>
          node3:
            name: node-3
            ip: <node-3 IP>

Let's see below, the content of the playbooks ``/etc/ansible/roles/wazuh-ansible/playbooks/wazuh-opendistro-kibana.yml``.

.. code-block:: yaml

    - hosts: <your opendistro host>
      roles:
        - role: ../roles/opendistro/opendistro-kibana
      vars:
        ansible_shell_allow_world_readable_temp: true

These files are designed to run the installations of each service individually.

Let's take a closer look at the content.

The first line ``hosts:`` indicates the machines where the commands below will be executed.

The ``roles:`` section indicates the roles that will be executed on the hosts.

2 - Preparing the playbook
--------------------------

We could configure these three files and execute them, but we are going to create a single file that executes the installation of the services in our Elastic Stack Server.

.. code-block:: console

	ansible@ansible:/etc/ansible/wazuh-ansible$ cat playbooks/wazuh-opendistro-and-kibana.yml

.. code-block:: yaml

> 	:class: output

Author: Done

.. Copyright (C) 2021 Wazuh, Inc.

.. meta::
  :description: Deploying the Open Distro for Elasticsearch server involves installing the Open Distro and Kibana services. Find out how to do it step by step in this section.

.. _wazuh_ansible_elk_server:

Install Open Distro for Elasticsearch Server
============================================

> The deployment of the Open Distro for Elasticsearch server involves the installation of Opendistro and Kibana services. In the repository of Ansible that Wazuh has we can find the playbooks and roles necessary to carry out the installation. The Ansible server must have access to the Open Distro for Elasticsearch server.

Reviewer: ```suggestion
The deployment of the Open Distro for the Elasticsearch server involves the installation of Opendistro and Kibana services. In the repository of Ansible that Wazuh has, we can find the playbooks and roles necessary to carry out the installation. The Ansible server must have access to the Open Distro for Elasticsearch server.
```

		│   ├── handlers
		│   ├── meta
		│   ├── tasks
		│   └── templates
		└── ansible-wazuh-manager
			├── defaults
			├── files
			├── handlers
			├── meta
			├── tasks
			├── templates
			└── vars

And we can see the preconfigured playbooks we have.

.. code-block:: console

	ansible@ansible:/etc/ansible/roles/wazuh-ansible$ tree playbooks/

.. code-block:: none
	:class: output

	playbooks/
	├── wazuh-agent.yml
	├── wazuh-elastic_stack-distributed.yml
	├── wazuh-elastic_stack-single.yml
	├── wazuh-elastic.yml
	├── wazuh-kibana.yml
	├── wazuh-manager-oss.yml
	├── wazuh-manager.yml
	├── wazuh-odfe-production-ready.yml
	├── wazuh-odfe-single.yml
	├── wazuh-opendistro-kibana.yml
	└── wazuh-opendistro.yml

Using **Opendistro** and **Kibana** roles, we will install and configure the Open Distro for Elasticsearch server components. There are several variables we can use to customize the installation or configuration. To consult the default configuration go to this :ref:`section <wazuh_ansible_reference>`.

If we want to change the default configuration, we can change the following files:
- ``/etc/ansible/roles/wazuh-ansible/roles/opendistro/opendistro-elasticsearch/defaults/main.yml``

> - ``/etc/ansible/roles/wazuh-ansible/roles/opendistro/opendistro-kibana/defaults/main.yml``

Reviewer 2: Fix this, this must be show in different lines

![image](https://user-images.githubusercontent.com/14913942/141313809-ab3b6a53-2e79-4cdb-8816-38001911c7af.png)

Author: Done
![image](https://user-images.githubusercontent.com/8108052/141551981-4682a4e3-49a3-4a85-97d5-bcd26dcf1d39.png)

.. Copyright (C) 2021 Wazuh, Inc.

.. meta::
  :description: Deploying the Open Distro for Elasticsearch server involves installing the Open Distro and Kibana services. Find out how to do it step by step in this section.

.. _wazuh_ansible_elk_server:

Install Open Distro for Elasticsearch Server
============================================

> The deployment of the Open Distro for the Elasticsearch server involves the installation of Opendistro and Kibana services. In the repository of Ansible that Wazuh has, we can find the playbooks and roles necessary to carry out the installation. The Ansible server must have access to the Open Distro for Elasticsearch server.

Reviewer 3: ```suggestion
The deployment of the Open Distro for the Elasticsearch server involves the installation of Open Distro for Elasticsearch and Kibana services. In the Wazuh Ansible repository, we can find the playbooks and roles necessary to carry out the installation. The Ansible server must have access to the Open Distro for Elasticsearch server.
```

- `1 - Access to wazuh-ansible`_
- `2 - Preparing the playbook`_
- `3 - Running the playbook`_

.. warning::

	In previous versions of this guide, playbooks were used pointing to roles to install Elasticsearch.

> 	Currently, the roles for the installation following guides from previous versions are preserved.

Reviewer 3: ```suggestion
	Currently, the roles for the installation following previous guides are preserved.
```

> 	Since version 4.2.4 the installation steps of this guide point to the roles that install Open Distro for Elasticsearch.

Reviewer 3: ```suggestion
	Starting Wazuh v4.2.5, the installation steps of this guide point to the roles that install Open Distro for Elasticsearch.
```

.. warning::

>     About the nomenclatures:

Reviewer 3: ```suggestion
    About the nomenclature:
```
